iceoryx_examples/iceperf/README.md

```diff
@@ -16,27 +16,29 @@ At the end of the benchmark, the average latency for each payload size is printe
 ## Run iceperf
 
 Create three terminals and run one command in each of them.
-The order is first the RouDi daemon, then iceperf-laurel which is the leader in this setup
-and then iceperf-hardy for doing the ping pong measurements with iceperf-laurel. 
-You can set the number of measurement iterations (number of roundtrips) with a command line paramter
-of iceperf-laurel (e.g. `./iceperf-laurel 100000`)
+The order is first the RouDi daemon, then `iceperf-bench` as follower and leader with the `-f` or
+`--follower` and `-l` or `--leader` command line parameter.
+In this setup the leader is doing the ping pong measurements with the follower.
+You can set the number of measurement iterations (number of roundtrips) with a command line parameter
+of iceperf-bench (e.g. `./iceperf-bench -l -n 100000`).
+There are further options which can be printed by `./iceperf-bench -h`.
 ```sh
     # If installed and available in PATH environment variable
     iox-roudi
     # If build from scratch with script in tools
     $ICEORYX_ROOT/build/install/prefix/bin/iox-roudi
 
-    build/iceoryx_examples/iceperf/iceperf-laurel
+    build/iceoryx_examples/iceperf/iceperf-bench -f
```

Comment:
Does the follower needs to be started always first? If so this should be mentioned.

Reply:
I refactored this code and now the order can be arbitrary